widgets/dock/manifest.json

```
@@ -0,0 +1,69 @@
+{
+  "name": "dock",
+  "label": "Dock Panel",
```

For consistency, should this just be called "Dock"?

```suggestion
  "label": "Dock",
```